.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+version: 2.1
+jobs:
+  test:
+    parallelism: 4
+    #run 4 test runs at the same time
+    working_directory: ~/circleci-python
+    docker:
+      - image: "circleci/python:3.9.5"
+    # uses python 3.9.5 
```

Comment:
We mentioned in our conversation a way to change the python version. That's this line here. Theoretically, you could have multiple python versions being tested at once, although that's a bit incompatible with the setup I have going on right now.